README.md

```diff
@@ -36,12 +36,6 @@ primer provides some the context for this project.
 
 ## Maintaining the Schema
 
-**New system requirement: [Mike Farah's GO-based yq](https://github.com/mikefarah/yq)**
-
-Some optional components use the Java-based [ROBOT](http://robot.obolibrary.org/) or Jena arq.
-Jena riot is also a part of the MongoDB dumping, repairing and validation workflow, if the user wishes
-to generate and validate RDF/TTL.
-
```

Comment:
This information is already in `MAINTAINERS.md`.

Reply:
Didn't you identify some additional system dependencies like pandoc?

Reply:
Hmm... regarding pandoc, specifically, I don't remember why I added pandoc to the Dockerfile in the first place. Given that I've never (knowingly) used pandoc before and given that the command that installs pandoc has been in the Dockerfile for as long as I can remember, I suspect I encountered an error message in my very early interaction with nmdc-schema saying it was missing (for example, while I was running `make` commands that ended up being irrelevant for me, such as `$ make setup`). I'll remove that installation step from the Dockerfile tomorrow if I don't remember why I added it in the first place.

Reply:
I think I know why I had installed pandoc — it was (and still is) listed in the dependency list in `MAINTAINERS.md`.

`MAINTAINERS.md` as of 7 months ago: https://github.com/microbiomedata/nmdc-schema/commit/e22539bbea80fa09269931f66672b0f2ffeae224

I don't know what part of the project depends on it, though. Maybe it is not a dependency anymore.

Reply:
It was added to the list back in 2021, here: https://github.com/microbiomedata/nmdc-schema/commit/7e291ca452b3e0f8dda8d7052eddca2d0dd24f52